### PR TITLE
add Test for SelectionSort

### PR DESCRIPTION
**Pull request by hnq-jan8**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [ ] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: src/test/java/com/thealgorithms/maths/SumOfDigitsTest.java [hunk 1]
```diff
@@ -1,25 +1,25 @@
-package com.thealgorithms.maths;
```
Comment (siriak) [inline src/test/java/com/thealgorithms/maths/SumOfDigitsTest.java:1]: Why change this file?

**Review by siriak — CHANGES_REQUESTED** [review]
New tests look good, but please revert unrelated changes to SumOfDigitsTest

File: src/test/java/com/thealgorithms/maths/SumOfDigitsTest.java (hunk 1 above)
Comment (hnq-jan8) [reply src/test/java/com/thealgorithms/maths/SumOfDigitsTest.java:1]: I thought that the file had some conflict I resolved it.
I have reverted it as your request

**Review by siriak — APPROVED** [review]
Thanks!

**Pull request merged**